include/rfb.js

```diff
@@ -2122,6 +2175,219 @@ var RFB;
+            common_XT_scancode["Delete"] = 0xE053;
+            common_XT_scancode["OSLeft"] = 0xE05B;
+            common_XT_scancode["OSRight"] = 0xE05C;
+            common_XT_scancode["ContextMenu"] = 0xE05D;
+            common_XT_scancode["BrowserSearch"] = 0xE065;
+            common_XT_scancode["BrowserFavorites"] = 0xE066;
+            common_XT_scancode["BrowserRefresh"] = 0xE067;
+            common_XT_scancode["BrowserStop"] = 0xE068;
+            common_XT_scancode["BrowserForward"] = 0xE069;
+            common_XT_scancode["BrowserBack"] = 0xE06A;
+            common_XT_scancode["NumpadComma"] = 0x007E;
+
+            var chromium_XT_scancode = {};
+            chromium_XT_scancode["F13"] = 0x005B;
+            chromium_XT_scancode["F14"] = 0x005C;
+            chromium_XT_scancode["F15"] = 0x005D;
+            chromium_XT_scancode["F16"] = 0x0063;
+            chromium_XT_scancode["F17"] = 0x0064;
+            chromium_XT_scancode["F18"] = 0x0065;
+            chromium_XT_scancode["F19"] = 0x0066;
+            chromium_XT_scancode["F20"] = 0x0067;
+            chromium_XT_scancode["F21"] = 0x0068;
+            chromium_XT_scancode["F22"] = 0x0069;
+            chromium_XT_scancode["F23"] = 0x006A;
+            chromium_XT_scancode["F24"] = 0x006B;
+            chromium_XT_scancode["Undo"] = 0xE008;
+            chromium_XT_scancode["Paste"] = 0xE00A;
+            chromium_XT_scancode["Cut"] = 0xE017;
+            chromium_XT_scancode["Copy"] = 0xE018;
+            chromium_XT_scancode["LaunchMail"] = 0xE01E;
+            chromium_XT_scancode["Eject"] = 0xE02C;
+            chromium_XT_scancode["Help"] = 0xE03B;
+
+            var gecko_XT_scancode = {};
+            gecko_XT_scancode["NumpadEqual"] = 0x0059;
+            gecko_XT_scancode["F13"] = 0x0064;
+            gecko_XT_scancode["F14"] = 0x0065;
+            gecko_XT_scancode["F15"] = 0x0066;
+            gecko_XT_scancode["F16"] = 0x0067;
+            gecko_XT_scancode["F17"] = 0x0068;
+            gecko_XT_scancode["F18"] = 0x0069;
+            gecko_XT_scancode["F19"] = 0x006A;
+            gecko_XT_scancode["F20"] = 0x006B;
+            gecko_XT_scancode["F21"] = 0x006C;
+            gecko_XT_scancode["F22"] = 0x006D;
+            gecko_XT_scancode["F23"] = 0x006E;
+            gecko_XT_scancode["F24"] = 0x0076;
+            gecko_XT_scancode["KanaMode"] = 0x0070;
+            gecko_XT_scancode["Lang2"] = 0x0071;
+            gecko_XT_scancode["Lang1"] = 0x0072;
+            gecko_XT_scancode["IntlRo"] = 0x0073;
+            gecko_XT_scancode["Convert"] = 0x0079;
+            gecko_XT_scancode["NonConvert"] = 0x007B;
+            gecko_XT_scancode["LaunchApp2"] = 0xE021;
+            gecko_XT_scancode["Power"] = 0xE05E;
+            gecko_XT_scancode["LaunchApp1"] = 0xE06B;
+            gecko_XT_scancode["LaunchMail"] = 0xE06C;
+            gecko_XT_scancode["MediaSelect"] = 0xE06D;
+
+            function checkIfRunningInSupportedBrowser() {
```

Comment:
When possible, we shouldn't use browser checks.  Is there a feature detection check that we can do elsewhere?  I'd also prefer if the feature detection were moved out of this function, so that someone can force-enable the feature if need be.